warehouse/legacy/api/json.py

```diff
@@ -312,6 +313,17 @@ def release_factory(request):
     return release
 
 
+def user_factory(request):
+    username = request.matchdict["username"]
+
+    try:
+        user = request.db.query(User).filter(User.username == username).one()
+    except NoResultFound:
+        return HTTPNotFound(headers=_CORS_HEADERS)
+
+    return user
```

Comment:
Can't this use the existing `UserFactory` instead?

https://github.com/pypi/warehouse/blob/35f9cacfa538e6849d11644e5d99d63d6efdb203/warehouse/accounts/models.py#L58-L66

Reply:
(See some existing examples of `UserFactory` usage in `routes.py` for reference.)